init.c

```diff
@@ -16,7 +16,7 @@ int checkAndCountListOfStreams(lua_State *L, THCState *state, int arg,
                                int device)
 {
   if (!lua_istable(L, arg)) {
-    THError("expecting table of device streams");
+    THError("expecting array of device streams");
```

Comment:
also FYI this stream/device wait API could now be fully implemented in Lua if we want to using cutorch.Event().
these were some error message changes I made a while ago. really I should have made this API use a table whose keys were streams instead to enforce uniqueness.